pkg/operator/ceph/cluster/osd/provision_spec.go

```diff
@@ -355,3 +354,57 @@ func (c *Cluster) provisionOSDContainer(osdProps osdProperties, copyBinariesMoun
 
 	return osdProvisionContainer, nil
 }
+
+// deleteAllOrphanedPrepareJobs deletes all osd prepare jobs which are bound to kubernetes nodes which no longer exist.
+func (c *Cluster) deleteAllOrphanedPrepareJobs() {
+	listOpts := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("%s=%s",
+			k8sutil.AppAttr, prepareAppName,
+		),
+	}
+	jobClientset := c.context.Clientset.BatchV1().Jobs(c.clusterInfo.Namespace)
```

Comment:
Could `c.context.Clientset.BatchV1().Jobs().List()` be used here instead of creating `jobClientset`?